packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/extractionservice/ExtractionStatusUpdate.java

```diff
@@ -0,0 +1,24 @@
+package software.uncharted.terarium.hmiserver.models.extractionservice;
+
+import lombok.Data;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+
+import java.util.UUID;
+
+@Data
+@TSModel
+public class ExtractionStatusUpdate {
+    private UUID documentId;
+    private Integer step;
+    private Integer totalSteps;
+    private String message;
+    private String error;
+
+    public ExtractionStatusUpdate(UUID documentId, Integer step, Integer totalSteps, String message, String error) {
```

Comment:
Is this constructor used? Not a deal breaker at all just curious why its here :)

Reply:
It is the only consrtructor used - ExtractionService L 272.